pkg/virt-launcher/virtwrap/agent-poller/agent_poller.go

```diff
@@ -486,6 +476,10 @@ func fetchAndStoreGuestInfo(infoTypes libvirt.DomainGuestInfoTypes, agentPoller
 	if infoTypes&libvirt.DOMAIN_GUEST_INFO_USERS != 0 {
 		agentPoller.agentStore.Store(libvirt.DOMAIN_GUEST_INFO_USERS, convertToUsers(guestInfo))
 	}
+
+	if infoTypes&FSInfoType != 0 {
```

Comment:
Using the constant here hides that this condition is different than those above. When the constant is expanded, the condition becomes:
```go
if infoTypes&libvirt.DOMAIN_GUEST_INFO_FILESYSTEM != 0 || infoTypes&libvirt.DOMAIN_GUEST_INFO_DISKS != 0 {}
```
Which is probably not what we want. I think we want both info types to be present, not just one of them.